usaspending_api/search/models/award_search.py

```diff
@@ -109,7 +109,33 @@ class AwardSearch(models.Model):
     covid_spending_by_defc = models.JSONField(null=True)
     total_covid_outlay = models.DecimalField(max_digits=23, decimal_places=2, blank=True, null=True)
     total_covid_obligation = models.DecimalField(max_digits=23, decimal_places=2, blank=True, null=True)
-
+    officer_1_amount = models.DecimalField(max_digits=23, decimal_places=2, blank=True, null=True)
+    officer_1_name = models.TextField(null=True)
+    officer_2_amount = models.DecimalField(max_digits=23, decimal_places=2, blank=True, null=True)
+    officer_2_name = models.TextField(null=True)
+    officer_3_amount = models.DecimalField(max_digits=23, decimal_places=2, blank=True, null=True)
+    officer_3_name = models.TextField(null=True)
+    officer_4_amount = models.DecimalField(max_digits=23, decimal_places=2, blank=True, null=True)
+    officer_4_name = models.TextField(null=True)
+    officer_5_amount = models.DecimalField(max_digits=23, decimal_places=2, blank=True, null=True)
+    officer_5_name = models.TextField(null=True)
+    is_fpds = models.BooleanField(default=False)
+    fpds_agency_id = models.TextField(null=True)
+    fpds_parent_agency_id = models.TextField(null=True)
+    latest_transaction_id = models.IntegerField(null=True)
+    base_and_all_options_value = models.DecimalField(max_digits=23, decimal_places=2, blank=True, null=True)
+    non_federal_funding_amount = models.DecimalField(max_digits=23, decimal_places=2, blank=True, null=True)
+    total_subaward_amount = models.DecimalField(max_digits=23, decimal_places=2, blank=True, null=True)
+    subaward_count = models.IntegerField(null=True)
+    base_exercised_options_val = models.DecimalField(max_digits=23, decimal_places=2, blank=True, null=True)
+    parent_award_piid = models.TextField(null=True)
```

Comment:
This field has an index on `awards`. It should probably get one on `award_search` as well